tests/acceptance/features/apiSharingNg/reshare.feature

```diff
@@ -0,0 +1,125 @@
+Feature: Reshare a share invitation
+  As a user
+  I want to be able to reshare the share invitations to other users
+  So that they can have access to it
+
+  https://owncloud.dev/libre-graph-api/#/drives.permissions/Invite
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+      | Carol    |
+
+
+  Scenario Outline: reshare a file to a user with different roles
+    Given user "Alice" has uploaded file with content "to share" to "/textfile1.txt"
+    And user "Alice" has sent the following share invitation:
+      | resourceType    | file               |
+      | resource        | textfile1.txt      |
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    When user "Brian" sends the following share invitation using the Graph API:
+      | resourceType    | file                       |
+      | resource        | textfile1.txt              |
+      | space           | Shares                     |
+      | sharee          | Carol                      |
+      | shareType       | user                       |
+      | permissionsRole | <reshare-permissions-role> |
+    Then the HTTP status code should be "200"
+    And for user "Carol" the space Shares should contain these entries:
+      | textfile1.txt |
+    Examples:
+      | permissions-role | reshare-permissions-role |
+      | Viewer           | Viewer                   |
+      | File Editor      | Viewer                   |
+      | File Editor      | File Editor              |
```

Comment:
```suggestion
      | permissions-role | reshare-permissions-role |
      | Viewer           | Viewer                   |
      | File Editor      | Viewer                   |
      | Viewer           | File Editor              |
      | File Editor      | File Editor              |
```
combination if needed?

Reply:
When a user receives a share as Viewer role, he/she cannot reshare the resource to other users with the role File Editor or higher roles. In this PR only positive case is covered. Negative cases will be covered in other PR.